src/main/java/com/karankumar/bookproject/backend/entity/Author.java

```diff
@@ -32,7 +38,7 @@
 @Data
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 @AllArgsConstructor
-@EqualsAndHashCode(callSuper = true)
+@EqualsAndHashCode(callSuper = true, exclude= "books")
 public class Author extends BaseEntity {
```

Comment:
@knjk04 To solve the requirement, 

I will have to include a hand-written constructor which takes only firstName and lastName as parameters. (books is going to be initialized with `new HashSet<>()`)

Please let me know if the approach is fine. I am not aware of any way to skip a particular field from `@AllArgsConstructor`

Reply:
That's fine.

Also, could you please resolve the merge confilcts?

Thanks!